Extending/Section-Trees/tree-actions.md

```diff
@@ -4,7 +4,7 @@ Items in an Umbraco Tree can have associated Actions, the actions visible to the
 
 You can set a User's permissions for each item in the Umbraco Content tree from the User Section of the Umbraco Backoffice.
 
-If you are developing a custom section, or a custom Dashboard, you might want to display some different options based on a Users's Permission set on a particular item.
+If you are developing a custom section, or a custom Dashboard, you might want to display some different options based on a User's permission set on a particular item.
```

Comment:
Forgive me for stupid question: but isn't it: ** on a Users permission set **

Doublechecked on https://www.merriam-webster.com/dictionary/%27s and it says that 's stands for IS, WAS, HAS, DOES.  None of the words match.

Reply:
Hi @dampee no problem with the question :)

That's not the only use of an Apostrophe.

Users = More than one user
User's = Something belongs to ONE user

E.g.

- There are many users of Umbraco.
- The user's computer had an issue today.

Or...

- The dog's bone was very tasty.
- The tree has many bananas.

If you wanted to say something belonged to me, because my name ends with an S you just add an apostrophe:

Chris' computer is orange like the original Umbraco logo.

Lots more detail can be found online, for example this page: [Grammar Book - Apostrophes](https://www.grammarbook.com/punctuation/apostro.asp)